base/interactiveutil.jl

```diff
@@ -251,7 +251,8 @@ function versioninfo(io::IO=STDOUT, verbose::Bool=false)
         println(io, "DEBUG build")
     end
     println(io,             "Platform Info:")
-    println(io,             "  System: ", Sys.KERNEL, " (", Sys.MACHINE, ")")
+    println(io,             "  OS: ", is_windows() ? "Windows" : is_apple() ?
```

Comment:
It would be helpful if we could print "Windows 8.1" or "macOS 10.6" etc.   Printing "Ubuntu 16.04" etc would be great, too, but I don't know if there is a good way to detect the distro?